boards/ek-lm4f120xl/Makefile.features

```diff
@@ -0,0 +1,5 @@
+FEATURES_PROVIDED += cpp
+#FEATURES_PROVIDED += periph_adc
```

Comment:
I have not completely Implemented it. 
Its missing ISR and haven't yet tested it as well.
I will include this periph support in the coming commits. Wanted to complete the initial commit first